CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 2.6.2 (2019-05-13)
```

Comment:
Can you remove the date and rename the commit as updating change log for upcoming release.

Reply:
@yogiraj07 this is done now.